### PR TITLE
use jaspColumn

### PR DESCRIPTION
**Pull request by JorisGoosen**
Uses jasp and jaspBase with changes from
https://github.com/jasp-stats/jaspBase/pull/138
https://github.com/jasp-stats/jasp-desktop/pull/5302

.setColumnDataAsScale etc shouldnt be used by analyses (anymore)

### CODE REVIEW

File: R/commonDiscoverDistributions.R [hunk 1]
```diff
@@ -31,14 +25,18 @@ gettextf <- function(fmt, ..., domain = NULL)  {
...
+    if(options[["newVariableName"]] != "")
+    {
+      jaspResults[['sampleColumn']] <- createJaspColumn(options[["newVariableName"]])
+      
+      didItWork <- switch(as,
+      scale=  jaspResults[['sampleColumn']]$setScale(  sample),
+      ordinal=jaspResults[['sampleColumn']]$setOrdinal(sample),
+              jaspResults[['sampleColumn']]$setNominal(sample))
+
+      if(!didItWork)
```
Comment (Kucharssim) [inline R/commonDiscoverDistributions.R:37]: I get
 
```r
This analysis terminated unexpectedly.

Error in !didItWork: invalid argument type
```

**Review by Kucharssim — COMMENTED** [review]
Thanks for doing this! However I get an error, see the code comment

File: R/commonDiscoverDistributions.R (hunk 1 above)
Comment (JorisGoosen) [reply R/commonDiscoverDistributions.R:37]: cause you first need those PRs

Comment (Kucharssim) [reply R/commonDiscoverDistributions.R:37]: Which are "those"?

Comment (Kucharssim) [reply R/commonDiscoverDistributions.R:37]: Ah I am an idiot, you linked them already here. oops 😅

**Review by Kucharssim — APPROVED** [review]
Ok, this does the job indeed!

**Pull request merged**